lib/seattleflu/id3c/cli/command/etl/__init__.py

```diff
@@ -133,6 +137,58 @@ def standardize_race(race):
     return list(map(standardize_race, races))
 
 
+def ethnicity(ethnicity: str) -> Optional[bool]:
+    """
+    Returns a standardized ethnicity based on FHIR values:
+    (https://www.hl7.org/fhir/v3/Ethnicity/cs.html)
+
+    >>> ethnicity("HISPANIC OR LATINO")
+    True
+
+    >>> ethnicity("NOT HISPANIC OR LATINO")
+    False
+    
+    >>> ethnicity("NULL") == None
+    True
+
+    A :class:`UnknownEthnicityError` is raised when an unknown value is encountered:
+
+    >>> ethnicity("FOOBARBAZ")
+    Traceback (most recent call last):
+        ...
+    seattleflu.id3c.cli.command.etl.UnknownEthnicityError: Unknown ethnicity value «foobarbaz»
+    """
+
+    if not ethnicity:
+        return None
+    else:
+        ethnicity = standardize_whitespace(ethnicity.lower())
+
+    # Leaving this code here to be implemented later. My original approach was to use FHIR 
+    # coding for ethnicity, which would be preffered, but for consistency with other ETLs
+    # I switched to ingesting ethnicity as a boolean. To transition to FHIR codes across all projects will
+    # require updating multiple ETLs, shipping views, and re-ingesting data. A card has been added
+    # to tackle this at a later date.
+    # -drr 2021-12-30
+
+    #hispanic_or_latino = create_coding("http://hl7.org/fhir/v3/Ethnicity", "2135-2", "hispanic or latino")
+    #not_hispanic_or_latino = create_coding("http://hl7.org/fhir/v3/Ethnicity", "2186-5", "not hispanic or latino")
+
+    mapper = {
+        "hispanic or latino":               True,           # hispanic_or_latino,
+        "not hispanic or latino":           False,          # not_hispanic_or_latino,
+        "white/caucasian":                  False,          # not_hispanic_or_latino,
```

Comment:
I'm not sure if this came from pre-existing code, but if we explicitly map this, I would code this as `None`, as I think its a race category, not ethnicity.

Reply:
> I'm not sure if this came from pre-existing code, but if we explicitly map this, I would code this as `None`, as I think its a race category, not ethnicity.

That came from the "Codebook" they provided us (which is an Excel document, not actually defined in REDCap):
https://seattle-flu-study.slack.com/archives/C0173B4S1LY/p1639175618340200?thread_ts=1638557927.266100&cid=C0173B4S1LY

It might be safe to just drop it value since it doesn't look like it has been used in REDCap based on Cooper's summary.

Reply:
Nod. We could confirm with the folks analyzing data downstream, like Amanda or Cecile, but based on my understanding, if we want this to be standardized across projects, we should just map that as `None`

Reply:
Removed that value from the map, since it's not present in REDCap.